src/ProjectTemplates/test/IdentityUIPackageTest.cs

```diff
@@ -26,59 +26,6 @@ public IdentityUIPackageTest(ProjectFactoryFixture projectFactory, ITestOutputHe
 
         public ITestOutputHelper Output { get; }
 
-        public static TheoryData<IDictionary<string, string>, string, string[]> MSBuildIdentityUIPackageOptions
```

Comment:
Bootstrap3 was removed, breaking this test, but nobody noticed because it was Quarantined.